talk/owt/sdk/base/cameravideocapturer.cc

```diff
@@ -1,8 +1,12 @@
+// Copyright (C) <2018> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
 /*
- *  Copyright (c) 2017 The WebRTC project authors. All Rights Reserved.
```

Comment:
Please confirm this file is created in 2013.

Reply:
They derive from webrtc/test/vcm_capturer.h|cc and those files are created in 2013. So it should be 2013.